mediathread/assetmgr/models.py

```diff
@@ -339,3 +340,30 @@ def __unicode__(self):
 
     class Meta:
         ordering = ['title']
+
+
+class JuxtaposeAsset(models.Model):
+    added = models.DateTimeField(auto_now_add=True)
+    modified = models.DateTimeField(auto_now=True)
+    author = models.ForeignKey(User)
+    course = models.ForeignKey(Course)
+    title = models.TextField()
+    spine = models.ForeignKey(SherdNote, blank=True, null=True)
+
+
+class JuxtaposeMediaItem(models.Model):
```

Comment:
Also, I will rename "Item" to something else. I appreciate the detailed feedback.